src/Components/Search/Previews/Grids/ArtistSearch/MarketingCollections.tsx

```diff
@@ -143,7 +145,7 @@ export const MarketingCollectionsPreviewFragmentContainer = createFragmentContai
       <Responsive>
         {({ xs, sm, md }) => {
           return (
-            <Subscribe to={[SearchBarState]}>
+            <SearchBarConsumer>
```

Comment:
I wonder if hooks can be used as a child? I _think_ so, but not totally sure. 

That, said, even if we couldn't do ^, we can flatten this thing out nicely between the consumer and responsive.

If we add an export [here](https://github.com/artsy/reaction/blob/master/src/Utils/Responsive/DeprecatedResponsive.tsx#L12) then it should be possible to use the responsive component like so: 

```js
import { ResponsiveComponents } from 'Utils/Responsive/DeprecatedResponsive'

const { xs, sm, md } = useContext(ResponsiveComponents) 
const { searchState } = useContext(SearchBarContext)

...
```